flask-app/static_data_api.py

```diff
@@ -0,0 +1,33 @@
+import json
+from typing import List
+
+import pandas as pd
+
+from models import County, Voivodeship
+
+COUNTIES_CSV_FILEPATH = "static_data/counties.csv"
```

Comment:
Either this or the directory name ('static') is wrong.

Reply:
Fixed in 69ef17b